README.md

```diff
@@ -1,3 +1,12 @@
 # Rethnet
 
-This repository is only used for issues at the moment. Please take a look at the [Rethnet GitHub Project](https://github.com/orgs/NomicFoundation/projects/3) and [Hardhat repository](https://github.com/NomicFoundation/hardhat) instead.
+**Rethnet** is an Ethereum development runtime library - written in Rust - with bindings for the Node API (Typescript).
+Its goal is to provide a performant API for developers tooling, such as an EVM debugger or state inspector.
+
+Rethnet finds its origins in Hardhat, but is a complete rewrite of our Hardhat Node Typescript code to Rust, incorporating all of the lessons we have learned over the years, and much more to come.
+As such, it exists within - an experimental branch of - the [Hardhat monorepo](https://github.com/NomicFoundation/hardhat/tree/rethnet/main/).
```

Comment:
> As such, it exists within - an experimental branch of - the [Hardhat monorepo](https://github.com/NomicFoundation/hardhat/tree/rethnet/main/).

The use of hyphens here feels weird to me. It's not clear to me whether you're going for emphasis or a parenthetical phrase; if either of those is right, consider italics or parenthesis, respectively.